### DISCUSSION TITLE
Allow deployment event trigger filtered by environment name

### DISCUSSION THREAD

**Discussion by jirikopecky** [community/community#67727] [Actions]
### Select Topic Area

Product Feedback

### Body

When specifying `on.deployment` trigger in workflow, there is no option to filter on target environment name, resulting in large number of deployments in 'Inactive' state as we can check the target environment only in job condition.

We have multiple workflows triggered by `deployment` event, each targeting specific environment and allowing to filter on trigger instead of on job will result in much more cleaner deployment log.

**Comment by jmvtrinidad** [comment]
and probably we also want a way to filter if its ActivityType, currently it is only `created`, it will be good to have `completed`
